docs/02-membership.md

```diff
@@ -0,0 +1,350 @@
+| [Jordan Hrycaj](https://github.com/mjfh/) | 1 | Nimbus | |
+| [Kim De Mey](https://github.com/kdeme/) | 1 | Nimbus | |
+| [Dmitry Shmatko](https://github.com/zilm13/) | 1 | Teku | |
+| [Enrico Del Fante](https://github.com/tbenr/) | 1 | Teku | |
+| [Gabriel Fukushima](https://github.com/gfukushima/) | 1 | Teku | |
+| [Mehdi Aouadi](https://github.com/mehdi-aouadi) | 1 | Teku | |
+| [Lucas Saldanha](https://github.com/lucassaldanha) | 1 | Teku | |
+| [Paul Harris](https://github.com/rolfyone/) | 1 | Teku | |
+| [Stefan Bratanov](https://github.com/StefanBratanov/) | 1 | Teku | |
+| [Alex Vlasov](https://github.com/ericsson49/) | 1 | TXRX | |
+| [Anton Nashatyrev](https://github.com/Nashatyrev/) | 1 | TXRX | |
+| [Mikhail Kalinin](https://github.com/mkalinin/) | 1 | TXRX | |
+| [Roberto Saltini](https://github.com/saltiniroberto/) | 1 | Dependable Distributed Systems (DDS) | |
+| [Chenyi Zhang](https://github.com/czhang-fm/) | 0.5 | Dependable Distributed Systems (DDS) | |
+
+*Note: Protocol Guild's [split](https://app.splits.org/accounts/0xd4ad8daba9ee5ef16bb931d1cbe63fb9e102ec10/) contract contains all the above members plus one [multisig](https://app.safe.global/balances?safe=eth:0x69f4b27882eD6dc39E820acFc08C3d14f8e98a99) used for entity expenses.*
+
+## 2.2 Membership Requirements
+
+All current members have been contributing:
+
+1. To one or more of the list of [eligible projects](https://protocol-guild.readthedocs.io/en/latest/01-eligibility.html).
+2. Continuously for **at least 6 months** ahead of inclusion. This work is expected to be ongoing (e.g. not a short-term or one-off project). To avoid removal from the current membership, any breaks in contribution must be shorter than 1 quarter (3 months). Beyond this length, the member should be moved to "Inactive" status until contribution resumes.
+3. In a roughly full-time capacity. Anything less receives a [partial weighting](https://protocol-guild.readthedocs.io/en/latest/02-membership.html#time-weight-formula).
+
+## 2.3 Adding a Member
+
+An existing member should make a PR at [this repo](https://github.com/protocolguild/documentation) which proposes an addition to the member list above, along with the accompanying info:
+
+- Name / Identifier
+- Affiliated project they work on which is eligible
+- Links to relevant eligible work, eg. GitHub, research
+- 3-4 sentence summary of their contributions
+
+See some examples here: [open](https://github.com/protocolguild/documentation/pulls) or [closed](https://github.com/protocolguild/documentation/pulls?q=is%3Apr+is%3Aclosed+label%3A%22add+member%22) PRs
+
+Discussion should be open for at least one week to give members time to review and discuss.
+
+Bias or conflicts of interest of the nominator should be disclosed, if they exist, e.g. where one is an advisor to the other’s side project. After one week of rough consensus, the PR should be merged or rejected.
+
+## 2.4 Removing a Member
```

Comment:
Consider adding a line here saying that removals should also open a PR on the same file?